ui/src/message_popup/questions.js

```diff
@@ -34,7 +34,21 @@ export const inquiryQuestions:[Question] = [
     studentIds: [4], 
     id: 101,
     text: `At the conclusion of your lesson plan for this challenge, you seed a group discussion by asking "What are you curious about related to photosynthesis?"  Hayin says "Why are plants green?"  What do you do?`,
-    encodedText: `[:i]At the conclusion of your lesson plan for this challenge, you seed a group discussion.[:u]What are you curious about related to photosynthesis?[:4s]Why are plants green?`,
+    encodedText: [
```

Comment:
Would `messages` be more direct than `encodedText`, now that the encoding is removed?